client/lib/user/utils.js

```diff
@@ -10,6 +10,16 @@ var debug = require( 'debug' )( 'calypso:user:utilities' ),
 var user = require( 'lib/user' )();
 
 var userUtils = {
+	getLoginUrl: function( redirect ) {
+		let url = config( 'login_url' );
+
+		if ( redirect ) {
+			url += '?redirect_to=' + encodeURIComponent( redirect );
```

Comment:
If we need to modify an existing query string, the [`qs`](https://www.npmjs.com/package/qs) package could be handy here. eg:


```js
import qs from 'qs';
// ...
const query = qs.parse( queryString );
query.redirect_to = encodeURIComponent( redirect );
return qs.stringify( query );
```